nat/factory_darwin.go

```diff
@@ -17,15 +17,13 @@
 
 package nat
 
-import "os/exec"
-
 // NewService returns fake nat service since there are no iptables on darwin
 func NewService() NATService {
 	return &servicePFCtl{
 		ipForward: serviceIPForward{
-			CommandEnable:  exec.Command("/usr/sbin/sysctl", "-w", "net.inet.ip.forwarding=1"),
-			CommandDisable: exec.Command("/usr/sbin/sysctl", "-w", "net.inet.ip.forwarding=0"),
-			CommandRead:    exec.Command("/usr/sbin/sysctl", "-n", "net.inet.ip.forwarding"),
+			CommandEnable:  []string{"/usr/sbin/sysctl", "-w", "net.inet.ip.forwarding=1"},
```

Comment:
Having command factory here would allow to test `servicePFCtl`

Reply:
Yeah, but is this critical? Perhaps it's about time we stopped abusing 'request changes' feature.